### ISSUE TITLE
What is the select button?

### ISSUE THREAD

**Issue by thehauntedmattress** [tavicu/homebridge-samsung-tizen#3]
So I really like that I can essentially turn on my PlayStation 4 using this plugin.  The way I achieve is by having a command hit KEY_SOURCE and then KEY_LEFT six times, and then KEY_RIGHT one time (my PS4 is my HDMI 1 source, so it is second after Live TV).  However, what key is the select button (the key on the remote that is centered in between KEY_UP, KEY_DOWN, KEY_LEFT, and KEY_RIGHT?  

Also I would really love it if I could always make sure that if I turn the TV off, that is always powers off, and that even if it was already off, it won't power on.  Thanks.

**Comment by thehauntedmattress** [comment]
Basically my turn off question is two parts.  I understand that in order to turn on the TV, the plugin sends a WOL command.  However this question is the same as issue #1.  

1. With other plugins regarding tizen TVs, if I set a HomeKit scene that turns the TV off, and the TV is already off, the TV will turn ON.  How do I ensure that if my TV is already off, and I set a Homekit scene that turns the TV off, that it will not turn on?
2. For my PlayStation 4 on command, is there a way I make it so that if the TV is off, that it can turn on, and then run the command by only using one switch?  Right now I would only think the command works if the TV is already on.

{"name": "SOURCE", "command": ["KEY_SOURCE", "KEY_LEFT*6", "KEY_RIGHT", "KEY_ENTER"]},

**Comment by tavicu** [comment]
Regarding 2, i will add an optional parameter to power on the tv if it's off before sending a command. But it will have a small delay between powering on and the command because the TV will not receive commands for about 2 seconds after it turned on.

I will do the change today and come back to you.

**Comment by thehauntedmattress** [comment]
Also you should add a comma after the MAC address line on your sample config text because if someone were to copy that text directly, it would yield an invalid JSON.

"platforms": [ {
        "platform": "SamsungTizen",
        "devices": [{
            "name": "Bedroom TV",
            "ip": "10.20.30.40",
            **"mac": "A0:B1:C2:D3:E4:F5",**
            "switches": [

**Comment by tavicu** [comment]
@thehauntedmattress Thank you. 
1. Changed the README.
2. I have looked over the KEY_HDMI1 and indeed it will not switch to HDMI even if the TV is able to power on the console. Right now the only option i can think of is KEY_SOURCE, switch to console, KEY_ENTER. Just like you did.
3. Released a new version with the option to turn the TV first. It will have a delay of 1500 ms before sending the commands.
```
{"name": "Command 1", "power": true, "command": "KEY_VOLUP"}
```
